src/main/kotlin/no/nav/familie/klage/behandling/BehandlingRepository.kt

```diff
@@ -44,4 +46,27 @@ interface BehandlingRepository : RepositoryInterface<Behandling, UUID>, InsertUp
             WHERE b.ekstern_behandling_id = :eksternBehandlingId AND f.fagsystem = :fagsystem"""
     )
     fun findByEksternBehandlingIdAndFagsystem(eksternBehandlingId: String, fagsystem: String): Behandling
```

Comment:
`findByEksternFagsakId...`

Reply:
Jeg tenker at det egentlige ikke trengs med riktige parametrar, 
Den engelske burde kun trenges når vi bruker spring-jdbc for å generere spørringen?